bin/search.js

```diff
@@ -51,13 +56,15 @@ function typeSearch(el) {
             }
         }
     }
+    var query = "";
     function createDataSource() {
         var local = JSON.parse(window.localStorage.getItem(localStorageDataKey)) || undefined;
         var bh = new Bloodhound({
             datumTokenizer: function (entry) {
                 return [entry.l, entry.p, entry.t].concat(entry.g).concat(entry.m);

```

Comment:
This is where we should be doing the work, actually. This function computes the list of tokens that constitute hits; we should be renormalizing the package names so that `foo-js`, `foo.js`, and `foojs` get an additional `foo` token